php/Dockerfile-7.2

```diff
@@ -89,7 +97,8 @@ RUN set -xe \
     && pecl clear-cache \
     && rm -Rf "$(pecl config-get temp_dir)/*" \
     && docker-php-source delete \
-    && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false $buildDeps
+    && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false $buildDeps \ 
+    && apt-get clean
```

Comment:
I might remember this wrong, but this might have been skipped on purpose so Docker-app does not need to start with `apt-get update -q -y` as well. However if that is a good thing or not is another question, it's just a break if this is removing update definitions, meaning it will need a version bump in format version string.

Reply:
You're probably right, the third commit was me going "Docker Hub reports that Plopix's PHP+Node is only 359 MB, wonder how he achieved that" (and copying stuff from https://github.com/Plopix/docker-php-ez-engine/blob/master/php72/Dockerfile to test). Interestingly it made no difference in image size, I'll revert that.